### PR TITLE
feat/add: demos +6

### PR DESCRIPTION
**Pull request by Dineshjnld**
## Related Issue(s)

[//]:
  #
  "[Cite any related issue(s) or feature request(s) from the repository's issue tracker.]"

Fixes # For #

## Proposed Changes

[//]: # '[List down the specific changes you have made in this pull request.]'

- [x]
- [x]

## Checklist

- [x] I have followed the repository's contribution guidelines.
- [x] I have added necessary documentation or updated existing documentation.
- [x] My code follows the established code style and formatting of the project.
- [x] I have tested my changes and they function as expected.
- [x] I have added suitable unit tests (if applicable).

## Screenshots
image

## To-Do

- [ ]

### CODE REVIEW

File: samples/demos/Naive-Bayes.zip
Comment (Nasfame) [inline samples/demos/Naive-Bayes.zip:1]: rename to naive-bayes.zip

File: samples/demos/inference-example.zip
Comment (Nasfame) [inline samples/demos/inference-example.zip:1]: rename more appropriate. This is too unclear

**Review by Nasfame — CHANGES_REQUESTED** [review]

File: samples/demos/winequality-red.zip
Comment (Nasfame) [inline samples/demos/winequality-red.zip:1]: rename: the demo is not clear

**Review by Nasfame — CHANGES_REQUESTED** [review]

File: samples/demos/Naive-Bayes.zip
Comment (Nasfame) [reply samples/demos/Naive-Bayes.zip:1]: @Dineshjnld any updates on this

**Comment by Nasfame** [comment]
@Dineshjnld sign the cla

**Comment by Nasfame** [comment]
@Dineshjnld update the branch

**Comment by Nasfame** [comment]
```
git fetch origin
git merge origin/main
```

**Comment by Nasfame** [comment]
Fixes #63

**Review by Dineshjnld — COMMENTED** [review]
done, but tests are failing?

**Review by Dineshjnld — COMMENTED** [review]
1

**Review by Dineshjnld — COMMENTED** [review]
.

**Review by Nasfame — APPROVED** [review]
LGTM

**Pull request merged**